### PR TITLE
Bump mosaicml/pytorch images to use new mosaicml/pytorch images with updated ubuntu 22.04

### PR DESCRIPTION
**Pull request by KuuCi**
# What does this PR do?

Bumps all images in composer to now use 22.04, follow up for https://github.com/mosaicml/composer/pull/3716

# What issue(s) does this change relate to?

https://databricks.atlassian.net/jira/software/c/projects/GRT/boards/4746?selectedIssue=GRT-3195

# Before submitting
- [x] Have you read the [contributor guidelines](https://github.com/mosaicml/composer/blob/main/CONTRIBUTING.md)?
- [ ] Is this change a documentation change or typo fix? If so, skip the rest of this checklist.
- [ ] Was this change discussed/approved in a GitHub issue first? It is much more likely to be merged if so.
- [x] Did you update any related docs and document your change?
- [x] Did you update any related tests and add any new tests related to your change? (see [testing](https://github.com/mosaicml/composer/blob/main/CONTRIBUTING.md#running-tests))
- [x] Did you run the tests locally to make sure they pass?
- [x] Did you run `pre-commit` on your change? (see the `pre-commit` section of [prerequisites](https://github.com/mosaicml/composer/blob/main/CONTRIBUTING.md#prerequisites))

### CODE REVIEW

**Review by snarayan21 — APPROVED** [review]
nice, lgtm. needs approval from a code owner though @mvpatel2000 @dakinggg

**Review by dakinggg — COMMENTED** [review]
Can you run the daily tests against this pr so we can be sure all the ci you are changing passes before merging?

**Review by dakinggg — APPROVED** [review]

**Pull request merged**
